Dockerfile

```
@@ -8,6 +10,9 @@ WORKDIR /app
 
 COPY package*.json ./
 
+RUN --mount=type=secret,id=NPM_REGISTRY \
+    cat /run/secrets/NPM_REGISTRY > ~/.npmrc
+
 RUN npm ci --ignore-scripts
```

Weiß nicht ob es einen Unterschied macht, aber sicher ist sicher :D 
```suggestion
RUN --mount=type=secret,id=NPM_REGISTRY \
    cat /run/secrets/NPM_REGISTRY > ~/.npmrc && \
    npm ci --ignore-scripts && \
    rm ~/.npmrc
```